2018-11-28 11:45:23,871 (17452/MainThread) pgcli.main INFO - No default pager found in environment. Using os default pager
2018-11-28 11:45:23,879 (17452/MainThread) pgcli.main ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.0.1/libexec/lib/python3.7/site-packages/pgcli/main.py", line 479, in connect\n    raise e\n  File "/usr/local/Cellar/pgcli/2.0.1/libexec/lib/python3.7/site-packages/pgcli/main.py", line 468, in connect\n    application_name=\'pgcli\', **kwargs)\n  File "/usr/local/Cellar/pgcli/2.0.1/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 197, in __init__\n    self.connect()\n  File "/usr/local/Cellar/pgcli/2.0.1/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 239, in connect\n    **kwargs)\n  File "/usr/local/Cellar/pgcli/2.0.1/libexec/lib/python3.7/site-packages/psycopg2/__init__.py", line 130, in connect\n    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)\n  File "/usr/local/Cellar/pgcli/2.0.1/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 40, in _wait_select\n    state = conn.poll()\npsycopg2.OperationalError: FATAL:  database "susannakosonen" does not exist\n\n'
2018-11-28 11:45:35,715 (17457/MainThread) pgcli.main INFO - No default pager found in environment. Using os default pager
2018-11-28 11:47:35,036 (17457/MainThread) pgcli.pgexecute ERROR - sql: 'SELECT * FROM (\n            SELECT student_interim_id,\n            event_type,\n            DENSE_RANK() OVER (\n              PARTITION BY student_interim_id\n              ORDER BY created_at DESC\n            ) AS event_number\n            FROM student_interim_events\n            LEFT JOIN student_interims ON (student_interims.id = student_interim_events.student_interim_id)\n            LEFT JOIN interims ON (interims.id = student_interims.interim_id)\n            LEFT JOIN rounds ON (rounds.id = interims.round_id)\n            WHERE rounds.id = 4\n          ) AS ranked_student_interims\n          WHERE event_number = 1 AND event_type = 1', error: ProgrammingError('column reference "created_at" is ambiguous\nLINE 6:               ORDER BY created_at DESC\n                               ^\n')
2018-11-28 11:47:35,037 (17457/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/2.0.1/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 357, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/2.0.1/libexec/lib/python3.7/site-packages/pgcli/pgexecute.py", line 389, in execute_normal_sql\n    cur.execute(split_sql)\npsycopg2.ProgrammingError: column reference "created_at" is ambiguous\nLINE 6:               ORDER BY created_at DESC\n                               ^\n\n'
2021-02-04 12:37:33,078 (43774/MainThread) pgcli.main INFO - No default pager found in environment. Using os default pager
2021-02-04 12:37:33,079 (43774/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
